src/covid_model_seiir_pipeline/pipeline/regression/data.py

```diff
@@ -190,6 +243,30 @@ def load_five_year_population(self, location_ids: List[int]) -> pd.DataFrame:
         population = population.loc[in_locations & is_2019 & is_both_sexes & is_five_year_bins, :]
         return population
 
+    def load_hospital_correction_data(self) -> 'HospitalCorrectionsData':
+        metadata = self.get_model_inputs_metadata()
+
+        model_inputs_path = Path(metadata['output_path'])
+        corrections_data = {}
+        file_map = (
+            ('hospitalizations', 'hospital_census'),
+            ('icu', 'icu_census'),
+            ('ventilators', 'ventilator_census'),
+        )
+        for dir_name, measure in file_map:
+            path = model_inputs_path / 'output_measures' / dir_name / 'population.csv'
+            df = pd.read_csv(path)
+            df['date'] = pd.to_datetime(df['date'])
+            df = df.loc[(df.age_group_id == 22) & (df.sex_id == 3)]
+            df = df[["location_id", "date", "value"]]
+            if df.groupby(["location_id", "date"]).count().value.max() > 1:
+                raise ValueError(f"Duplicate usages for location_id and date in {path}")
+            # Location IDs to exclude from the census input data. Requested by Steve on 9/30
+            census_exclude_locs = [200, 69, 179, 172, 170, 144, 26, 74, 67, 58]
```

Comment:
Might be worth checking with Steve on these locations, he gave no indication that they'd change ever but that was a few months ago.